test/Libraries/DynamoPythonTests/CodeCompletionTests.cs

```diff
@@ -12,7 +11,7 @@
 namespace DynamoPythonTests
 {
     [TestFixture]
-    internal class SharedCodeCompletionProviderTests : CodeCompletionTests
```

Comment:
whats the benefit of this change?

Reply:
tests seemed to be identified (and run .. no sure though) twice
There was no benefit for deriving from the CodeCompletionTests class